packages/react-tinacms-editor/src/core/context/useProsemirror.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+
+Copyright 2019 Forestry.io Inc
```

Comment:
We're in 2020 now :)

Reply:
Unrelated, but I'm just noticing this copyright isn't very flexible.

Reply:
Legit, rope @jpatters in on that